app/src/main/res/layout/item_bill.xml

```diff
@@ -20,77 +21,99 @@
 
     </data>
 
-    <LinearLayout
+    <android.support.v7.widget.CardView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:gravity="center"
-        android:orientation="horizontal"
-        android:padding="@dimen/dp_5">
+        android:layout_marginBottom="@dimen/dp_8"
+        car_view:cardElevation="@dimen/dp1"
+        >
 
         <LinearLayout
-            android:layout_width="0dp"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="@dimen/dp_10"
-            android:layout_marginRight="@dimen/dp_10"
-            android:layout_weight="1"
-            android:background="@color/color_white"
+            android:gravity="center"
             android:orientation="horizontal"
-            android:padding="@dimen/dp_10">
+            >
 
             <TextView
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/dp_30"
-                android:layout_marginRight="@dimen/dp_30"
-                android:text="@{item.serviceName}"
-                android:textColor="@color/color_black"
-                android:textSize="@dimen/sp_10"/>
+                android:background="@color/colorPrimary"
+                android:padding="@dimen/dp_2"
+                android:text="@{position + 1 + ``}"
+                android:textColor="@color/color_white"
+                android:textSize="@dimen/sp_18"/>
 
-            <TextView
-                android:layout_width="wrap_content"
+            <LinearLayout
+                android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/dp_30"
-                android:layout_marginRight="@dimen/dp_30"
-                android:text="@{item.stylistName}"
-                android:textColor="@color/color_black"
-                android:textSize="@dimen/sp_10"/>
+                android:layout_marginEnd="@dimen/dp_10"
+                android:layout_marginRight="@dimen/dp_10"
+                android:layout_weight="1"
+                android:background="@color/color_white"
+                android:orientation="horizontal"
+                android:paddingLeft="@dimen/dp_5">
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:paddingStart="@dimen/dp_5"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)